src/components/subscriptions/TabContentTable.jsx

```diff
@@ -19,6 +21,7 @@ import {
   TAB_PENDING_USERS,
   TAB_REVOKED_USERS,
 } from './constants';
+import AddUsersButton from './AddUsersButton';
```

Comment:
Per the mocks, we should rename `AddUsersButton` to `InviteLearnersButton` and change the button text to "Invite learners" from "Add Users". This applies to both places where this component is now used.